tool/yubico-piv-tool.c

```diff
@@ -2588,6 +2612,23 @@ int main(int argc, char *argv[]) {
           ret = EXIT_FAILURE;
         }
         break;
+      case action_arg_moveMINUS_key: {
+        int from_slot = get_slot_hex(args_info.slot_arg);
+        int to_slot = get_slot_hex((enum enum_slot) args_info.to_slot_arg);
+        if (move_key(state, from_slot, to_slot) == false) {
+          ret = EXIT_FAILURE;
+        } else {
+          fprintf(stderr, "Successfully moved key.\n");
+        }
+        break;
+      }
```

Comment:
Should this case really have curly braces around it?

Reply:
It starts with an assignment. Although I am thinking of moving the two get_slot_hex() calls to move_key() call but wouldn't that make the line too long?

Reply:
Ah, that makes sense.